models/articles.model.js

```diff
@@ -12,3 +12,20 @@ exports.getArticlesModel = () => {
   `;
   return db.query(SQL).then(({ rows }) => rows);
 };
+
+exports.getArticlesByIdModel = (article_id) => {
+  let SQL = `SELECT * FROM articles 
+  `;
+  return db
+    .query(SQL)
+    .then(({ rows: articleList }) => {
+      if (article_id > articleList.length && article_id <= 2147483647) {
```

Comment:
This need changing please. 
The Id could be larger than the array length. If you delete an item, it doesn't replace the id, so you could have n original array of 10 things, delete 9, then add another. 
The next id would be 11, but the array length is 1.

You can use the query below to check is the id is in there. It will give you an empty array if not